webextension/js/popup.js

```diff
@@ -101,6 +101,28 @@ let Logic = {
     }
     return this._currentIdentity;
   },
+
+  generateIdentityName() {
+    const defaultName = 'Container';
+    let ids = [];
+
+    // This loop populates the 'ids' array with all the already-used ids.
+    this._identities.forEach(identity => {
+      if (identity.name.startsWith(defaultName)) {
+        let id = parseInt(identity.name.substr(defaultName.length))
```

Comment:
Nit: Add `radix` argument? http://eslint.org/docs/rules/radix
Probably only an issue if the `name` starts with a `0` though, so possibly pretty edge case.